nodestream_github/client/githubclient.py

```diff
@@ -240,6 +271,12 @@ async def _get_paginated(
             query_params.update(params)
 
         while url is not None:
+            if "&page=100" in url:
+                logger.warning(
+                    "The GithubAPI has reached the maximum page size "
+                    "of 100. The returned data may be incomplete."
+                )
```

Comment:
can we log the baseurl here?  Also, I don't think there's a test for this. Adding a new test that uses the pagination test as an example (either have it do 100 pages, or have it skip to the end?) in tests/client/test_githubclient.py is probably the way to go here.